spec/factories/role.rb

```diff
@@ -3,5 +3,6 @@
   factory :role do
```

Comment:
Scope models into Treelify module. In that way, you will avoid conflicts with existing models with the same name in other projects that will use this gem. Also, add prefix in db tables.